### PR TITLE
Mint Sign Up Form Ready For Review

### PR DESCRIPTION
**Pull request by Tripple-A**
### Description:
* Html form
### Acceptance Criteria
* Passed all Linting tests
* Passed validation

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# pull request under review by Addo

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/all.css">
+    <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/footer.css">
+</head>
+
+<body>
+    <header id="header">
```
Comment (Forison) [inline index.html]: Use class instead of id 🙂

File: index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        <div id="left-header">
+            <div id="question">Already have an Intuit account?</div>
+            <div><a href="#">I forgot my User ID or Password</a></div>
+        </div>
+        <button id="sign-up">
+            <i class="fa fa-lock"></i> Sign In
+        </button>
+    </header>
+    
+    <main id="main">
```
Comment (Forison) [inline index.html]: Use class instead of id 🙂

File: index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <div id="signup-box">
```
Comment (Forison) [inline index.html]: Use class instead of id 🙂 more

**Review by Forison — CHANGES_REQUESTED** [review]
Good Job @AHEEBA && AJIBADE, 🙂
I love the way you organized your files. Your project looks good. 👌
However, technically, your project violates some CSS best practices.

**[REQUIRED]**
Do not use id as selectors in your project(Id selectors have priority over other kinds of selectors. This can make it harder to add new rules using less specific selectors, leading to specificity wars), use class instead. read on the best practices [here](https://speckyboy.com/good-bad-css-practices/). 

**[OPTIONAL]**
> I saw you use ``` p ``` as wraps for you form element thus ```<label> and <input>``` I suggest you use ```<div>```. I understand they are block elements part ```div``` is more semantic and mostly use for this purpose
## Kindly make changes and submit them for review again. You are almost there.🙂
#### Happy coding 🙂🙂

**Review by SunnyAsar — APPROVED** [review]
# Hello @Tripple-A  and @raheebwa 👋 
Nicely done Guys, Let's move to the next project 🚀 
## You May Pass 👍

**Pull request merged**